src/scripts/migrateProjects.js

```diff
@@ -0,0 +1,472 @@
+/**
+ * name : migrateProjects.js
+ * author : Priyanka Pradeep
+ * created-date : 24-Aug-2024
+ * Description : script to create the project from consumption side.
+ */
+
```

Comment:
@priyanka-TL - Few questions?

We should use the org id of the creator for org id

Do we store the reference of mongo id in SCP?

Do we upload the JSON in cloud during migration?

Reply:
@aks30  Creator org Id is not available in template, Yes we are published_id. Yes uploading to cloud